BCStudentSoftwareDevTeam/celts
# Delete Recurring Events

Did
 - Wrote a deleteAllRecurringEvents function that deleted the event we are on and the recurring events after it.
 - Added a button in the modal that appears when the "Delete" button is pressed on edit event page.
 - Added a test suite for the deleteAllRecurringEvents function

Test
 - Run test
 - Create a recurring event.
 - Delete one of the recurring events by pressing the "delete" button on the edit event page. When modal pops up, click "Delete All Recurring" button. The event that you were on and all the recurring events after should be deleted.
